apps/user/views.py

```diff
@@ -80,8 +82,9 @@ class ProfileChart(APIView):
   def get(self, request, exam_id):
     """Sends attempts data to frontend for charts."""
     show_entries = 50
-    attempts = ExamAttempt.objects.filter(
-        exam=exam_id, user_id=request.user.id)[:show_entries]
+    attempts = ExamAttempt.objects.filter(exam=exam_id,
+                                          user_id=request.user.id,
```

Comment:
Using `user_id=request.user_id` would be more performant.
And `user=request.user` would be more consistent.